modules/disabling-etcd-encryption.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * authentication/encrypting-etcd.adoc
+
+[id="disabling-etcd_{context}"]
+= Disabling etcd encryption
+
+You can disable encryption of etcd data in your cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Modify the API server object:
++
+----
+$ oc edit apiserver
+----
+
+. Set the `encryption` field type to `identity`:
++
+[source,yaml]
+----
+spec:
+  encryption:
+    type: identity
+----
+
+. Save the file to apply the changes.
++
+The decryption process starts. It may take 20 minutes or longer for this process to complete, depending on the size of your cluster.
```

Comment:
I'd like to document https://bugzilla.redhat.com/show_bug.cgi?id=1775554 more detailed, e.g. time for revisions of restarts is needed, time for real migration is then consumed depending on the size of your cluster.

Reply:
:+1: